src/rigid.jl

```diff
@@ -81,7 +81,8 @@ function rigid_mm_slow(params, fixed, moving, thresh, SD; initial_tfm=IdentityTr
     return ratio(mm, thresh, Inf)
 end
 
-function qd_rigid_coarse(fixed, moving, mxshift, mxrot, minwidth_rot, SD;
+function qd_rigid_coarse(fixed, moving, mxshift, mxrot, minwidth_rot;
+                         SD=diagm(ones(ndims(fixed))),
```

Comment:
Any reason not to use `I`?

Reply:
No reason other than it didn't seem to work, and I don't know how to use `I`, so I switched to what I knew.